src/extension.ts

```diff
@@ -166,6 +166,9 @@ export async function activate(context: ExtensionContext) {
     new Disposable(() => Disposable.from(...disposables).dispose())
   );
 
+  //See https://github.com/JohnstonCode/svn-scm/issues/561
+  (process as any).noAsar = true;
```

Comment:
Do not do it

Reply:
The trick: https://github.com/JohnstonCode/svn-scm/blob/master/src/svn.ts#L10 not work when `noAsar` is true

Reply:
What deps is that trick bringing in again?

Reply:
To reduce size of ".vsix", some dependencies are removed, to use the embed module in "VSCode"

Reply:
See #409